cypress/integration/sitemap-vrt/index.spec.ts

```diff
@@ -0,0 +1,83 @@
+// ℹ️ If we use anon functions instead of arrow functions, we can leverage Mocha's context and pull the test name directly from this
+describe('Full Site VRT', function () {
+  describe('Visual regression tests', () => {
+    it('vrt homepage', () => {
```

Comment:
question: did you manually generate these or did you use some kind of script? We will have to add to these if we add/remove/change any page routes right?

Reply:
OK, my experiment failed. Sticking to this for now

Reply:
Thought: Can we make an object of pages/urls and then loop through that? The sidebar navigation tests do this so I didn't have to write really repetitive tests.